### DISCUSSION TITLE
Support for Java SDK 23?

### DISCUSSION THREAD

**Discussion by jmric** [forcedotcom/salesforcedx-vscode#5964] [Ideas]
Hey team, I was wondering if support for Java SDK 23 is underway or planned? 
I'm running latest SF CLI and VSCode plugins but can't launch Apex Language Server due to Java version. Need to be on this Java version because of our organisations cyber security policies.

PS C:\temp\_salesforce_v2-1> java --version
java 23.0.1 2024-10-15
Java(TM) SE Runtime Environment (build 23.0.1+11-39)
Java HotSpot(TM) 64-Bit Server VM (build 23.0.1+11-39, mixed mode, sharing)
PS C:\temp\_salesforce_v2-1> sf version    
(node:23968) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
@salesforce/cli/2.67.7 win32-x64 node-v22.11.0

**Comment by daphne-sfdc** [comment]
Hi @jmric, Thank you for this feature request.  We will be adding support for the next LTS version Java 25 once it is GA in September.
